modules/shared/src/main/scala/org/tessellation/scalacheck/generators/trust.scala

```diff
@@ -0,0 +1,64 @@
+package org.tessellation.scalacheck.generators
+
+import org.tessellation.schema.TrustValueRefinement.TrustValueRefinement
+import org.tessellation.schema.peer.PeerId
+import org.tessellation.schema.trust._
+import org.tessellation.security.hex.Hex
+
+import eu.timepit.refined.auto._
+import eu.timepit.refined.refineV
+import org.scalacheck.Arbitrary.arbitrary
+import org.scalacheck.{Arbitrary, Gen}
+
+object trust {
+
+  implicit val arbitraryHexString: Arbitrary[String] = Arbitrary {
+    Gen.choose(0, 0xffffff).flatMap(_.toHexString)
+  }
+
+  implicit val arbitraryTrustDouble: Arbitrary[Double] = Arbitrary {
+    Gen.choose(-1.0, 1.0)
+  }
+
+  implicit val arbitraryOptionalTrustValue: Arbitrary[Option[Double]] = Arbitrary {
+    Gen.option(Gen.choose(-1.0, 1.0))
+  }
```

Comment:
you should reuse the existing arbitrary - `arbitraryTrustDouble` (simply map it or lift into option). please don't repeat the same logic

Reply:
I intended to remove `arbitraryTrustDouble` but I forgot. It has been removed.